### ISSUE TITLE
Windows Desktop Client upload failure with 7mb files

### ISSUE THREAD

**Issue by CSchaefer70** [nextcloud/desktop#715]
While the web browser portion is working great uploading and downloading files. I can’t seem to get the desktop sync client to work properly on my Windows 8 or 10 with larger files (bigger then ~7mb?); not the file type.  I keep getting the error server replied: Not Acceptable.

### Expected behaviour
When the Windows desktop sync client is installed I should be able to sync files from a local folder to the NextCloud application provided I am within the file size and type parameters in the NextCloud configuration.

### Actual behaviour
Small files (up to about 7 mb) work correctly uploading and syncing correctly.  Larger files fail with "...server replied: Not Acceptable."

### Steps to reproduce
1. install nextcloud (in this case 14.0.2)
2. install desktop client (in this case Version 2.3.3 (build 1))
3. save a file over 7mb to a local sync folder

### Client configuration
Client version:
Version 2.3.3 (build 1)
Operating system:
Windows 8.1
OS language:
English
Qt version used by client package (Linux only, see also Settings dialog):

Client package (From Nextcloud or distro) (Linux only):

Installation path of client:
C:\Program Files (x86)\Nextcloud

and C:\Users\admin\AppData\Local\Nextcloud

### Server configuration
Shared server (Bluehost)

Operating system:
Linux
Web server:

Hosting Package | PlatinumBasic
-- | --
Server Name | box5474
cPanel Version | 70.0 (build 59)
Apache Version | 2.4.35
PHP Version | 7.0.32
MySQL Version | 5.6.41-84.1
Architecture | x86_64
Operating System | linux
Shared IP Address | 162.241.217.99
Path to Sendmail | /usr/sbin/sendmail
Path to Perl | /usr/bin/perl
Perl Version | 5.10.1
Kernel Version | 3.10.0-693.11.6.1.ELK.el6.x86_64

Nextcloud version:
14.0.2 (but happens on previous versions like 13 or 12)
Storage backend (external storage):
on same server
### Logs

10/11/2018 5:01:33 PM,    xxx.pdf,D:\archive xprojects\NDL360Life\nextcloudsyncfolder,The item is not synced because of previous errors: Error transferring https://ndl360life.com/nc/remote.php/dav/uploads/admin/2000696326/00000000 - server replied: Not Acceptable

```Template for output < 10 lines```

1. Client logfile: 
see above... "Server replied: Not Acceptable"

2. Web server error log:
No error

3. Server logfile: nextcloud log (data/nextcloud.log):
No error

**Comment by CSchaefer70** [comment]
Confirmed the problem on Windows 7, 8, 8.1 and 10 on different laptops.  Also attempted to change the default "nextcloud.cfg" chunkSize	5242880 to double 10485760 with no change... still the upload failure.

I also tried to compress using WinZIP into 4mb size for a 100mb file... works correctly: each chunk uploads without failure... so this must be a cache size,chunk network issue in Windows?  I don't know where else to check?

Android and at least three browsers (firefox, ie, chrome) seems to work well with large files and actually everything I use.

Any ideas or suggestions?

**Comment by camilasan** [comment]
Hi, did you try one of the daily builds (with a test account)? 
https://download.nextcloud.com/desktop/daily/Windows/

**Comment by CSchaefer70** [comment]
Thx for the reply!  No, but the version I am using suggests no upate available.  I will try the "daily" to see if anything changes and report back.  

However, I did play with the "chunksize" in the cfg file (C:\Users\admin\AppData\Local\Nextcloud\nextcloud.cfg) and modified the general settings to include "chunksize=209715520" or 4 times the default and everything seems to be working:

[General]
confirmExternalStorage=true
newBigFolderSizeLimit=250
optionalDesktopNotifications=true
useNewBigFolderSizeLimit=false
crashReporter=false
monoIcons=false
chunkSize=20971520

I have not closed because I wanted to test with other devices to confirm the windows failure only.

**Comment by maghog** [comment]
Tested to sync a 40 MB file on Windows 10 with client 2.5.0 daily (build 20180927) and it works.

**Comment by CSchaefer70** [comment]
I really appreciate the feedback - THANK YOU!!!!!!!!

Interesting stuff... I tried the daily WindowsDesktopClient [Nextcloud-2.5.0.61464-daily-20181016] and could never quite get it stable (Windows 8.1) with lots of "connection time out" messages.  After installation I could not find the "cfg" file in "appdata...local" to compare.  The daily client does upload small files without failures but seems to get lost/timeout with larger (eg 30mb).  

I removed the "daily" and reinstalled the "Official Stable v2.3.3".  Upon re-installation, the old profile configuration was gone and required data entry as if a new install... URL, Password, etc.  Once configured virgin it failed until I forced the "chunkSize" in the general section of the "cfg" file.  Unless someone wants me to confirm a file size on a specific build on a specific platform I'll stick with "Version 2.3.3 build 1" with the mod to the nextcloud.cfg.... which I increased to allow 100mb files (setting "chunkSize=167772160").  The solution works for me on all Windows platforms I have for now: Win10, 8.1.
